computing_surface_normals.py

```diff
@@ -0,0 +1,70 @@
+import numpy as np
+from scipy.spatial import KDTree
+from scipy.linalg import eigh
+from myploty import plot_cloud_with_normals_plotly, plot_cloud_with_normals
+from load_point_cloud import load_point_cloud
+
+
+def normalize_point_cloud(point_cloud):
+    """
+    Normalize the point cloud so that longitude, latitude, and altitude are on a similar scale.
+    :param point_cloud: Nx3 numpy array where columns are longitude, latitude, and altitude.
+    :return: Normalized point cloud.
+    """
```

Comment:
Double check if it is suitable to update all the latitude, longitude, attitude values.